packages/slice-machine/server/src/api/index.ts

```diff
@@ -209,11 +210,11 @@ router.get(
   ): Promise<Express.Response> {
     // eslint-disable-next-line @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call
     const payload = await pushCustomType(req);
```

Comment:
same

Reply:
Same as slices, a big try catch inside the function itself